send-to-bluetooth@asgharalitaj/info.json

```diff
@@ -1,3 +1,3 @@
 {
-    "author": "asgharalitaj"
+    "author": "Asghar Ali Taj"
```

Comment:
This should be your GitHub username.

Reply:
I think i already changed that.

Reply:
Did you push up your changes? The content online is showing this current value here.

Reply:
yes, It's even showing in my repo.

Reply:
it is also showing in this very page, last commit "spelling changes"

Reply:
Your GitHub username does not have spaces in it. The latest change does. This field is for your GitHub username.

Reply:
![Screenshot from 2024-07-19 19-08-48](https://github.com/user-attachments/assets/9022b2e4-03b1-4400-ad15-01941518350f)

Reply:
in that latest spelling changes, I fixed that issue i think.

Reply:
Click on the fourth tab "Files changed" and make sure it is all commits and then scroll to the file in question.

Reply:
yes, I wanted spaces so i add them there.

Reply:
Again, that field is for your GitHub username. Your GitHub username does not have spaces in it.

Reply:
oh, it is a requirement, I didn't know that. I fixed it.